apache/trafficcontrol
# Add sort by privilege level to TO and use it in TP

* Add support for sorting roles by priv_level
* Change TP to request roles ordered by priv_level
* Add support to TO Go to order by descending order
* Change TP to request roles in descending order
* **DOES NOT INCLUDE DOCS - requires PR #3740**

## What does this PR (Pull Request) do?
- [x] This PR fixes #3226

## Which Traffic Control components are affected by this PR?
- Traffic Ops
- Traffic Portal

## What is the best way to verify this PR?
- Run the TO API tests and ensure they pass
- Build and start TO and TP, open the Roles section of TP, and ensure the roles are listed in descending order by privilege level

## The following criteria are ALL met by this PR
